<?xml version="1.0"?>
<launch>
	<node pkg="rosbag" type="play" name="rosbag" args="-d 2 -r .25 /home/eva/grasp_results/video_compare/robot_scene_capture.bag" output="screen"/>
	<node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
		<remap from="image" to="/kinect2/qhd/image_color_rect/decomp"/>
	</node>

	<node name="decompress" pkg="image_transport" type="republish" args="compressed in:=/kinect2/qhd/image_color_rect raw out:=/kinect2/qhd/image_color_rect/decomp" />
</launch>
